Source/WebKit/UIProcess/ios/WKContentView.mm

```diff
@@ -965,22 +992,71 @@ - (NSUInteger)_wk_pageCountForPrintFormatter:(_WKWebViewPrintFormatter *)printFo
     }
 
     if (!pageCount)
+        return nil;
+
+    auto attributes = adoptNS([[_WKPrintFormattingAttributes alloc] initWithPageCount:pageCount frameID:frameID printInfo:printInfo]);
+    return attributes;
+}
+
+- (NSUInteger)_wk_pageCountForPrintFormatter:(_WKWebViewPrintFormatter *)printFormatter
+{
+    if (printFormatter._shouldDrawUsingBitmap)
+        [self _waitForDrawToImageCallbackForPrintFormatterIfNeeded:printFormatter];
+    else
+        [self _waitForDrawToPDFCallbackForPrintFormatterIfNeeded:printFormatter];
```

Comment:
Do we need an `if` here? The methods are "if needed", so they should do nothing if they don't apply?